Library/ViewModels/PledgePaymentMethodsViewModel.swift

```diff
@@ -53,8 +53,7 @@ public final class PledgePaymentMethodsViewModel: PledgePaymentMethodsViewModelT
       }
 
     self.applePayStackViewHidden = configureWithValue
-      .map { ($0.project, $0.applePayCapable) }
-      .map(showApplePayButton(for:applePayCapable:))
+      .map { showApplePayButton(for: $0.project) }
```

Comment:
Hmmm I think we actually still need an `applePayCapable` value, because we need to know whether the device itself supports Apple Pay at all. The way we can determine this is by using `PKPaymentAuthorizationViewController.applePayDevice()`, which would passed in here: https://github.com/kickstarter/ios-oss/blob/master/Kickstarter-iOS/Views/Controllers/PledgePaymentMethodsViewController.swift#L170 .

Additionally, we might also want to include this behavior on `DeprecatedRewardPledgeViewController`, so that it's consistent between the two experiences: https://github.com/kickstarter/ios-oss/blob/master/Kickstarter-iOS/Views/Controllers/DeprecatedRewardPledgeViewController.swift#L67

We should also be able to remove all of the custom logic in `PKPaymentMethodsViewController+Helpers` now that we're only using `.applePayDevice()` and no longer care about which project is being passed in.

Reply:
Will check w/ product to make sure we want to include that on the `DeprecatedRewardPledgeViewController` and then file a separate ticket/PR. Removing the custom logic can also go in that separate ticket/PR.